nagios/README.md

```diff
@@ -9,36 +9,86 @@ service_perfdata_file_template=DATATYPE::SERVICEPERFDATA\tTIMET::$TIMET$\tHOSTNA
 
 service_perfdata_file_mode=a
 service_perfdata_file_processing_interval=15
-service_perfdata_file_processing_command=wavefront_perf_service
+service_perfdata_file_processing_command=wavefront-perf-service
 
 host_perfdata_file=[PATH_PERFDATA]/host-perfdata
 host_perfdata_file_template=DATATYPE::HOSTPERFDATA\tTIMET::$TIMET$\tHOSTNAME::$HOSTNAME$\tHOSTPERFDATA::$HOSTPERFDATA$\tHOSTCHECKCOMMAND::$HOSTCHECKCOMMAND$\tHOSTSTATE::$HOSTSTATE$\tHOSTSTATETYPE::$HOSTSTATETYPE$
 
 host_perfdata_file_mode=a
 host_perfdata_file_processing_interval=15
-host_perfdata_file_processing_command=wavefront_perf_host
+host_perfdata_file_processing_command=wavefront-perf-host
 ```
 
+#### Using CSP API Token
 ```
 define command {
-    command_name  wavefront_perf_host
-    command_line  /opt/nagios/etc/wf/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
+    command_name  wavefront-perf-host
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/host-perfdata \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_api_token [CSP_API_TOKEN] \
+                                                           --csp_base_url [CSP_BASE_URL]                                                         
+}
+
+define command {
+    command_name  wavefront-perf-service
+    command_line /usr/local/nagios/libexec/nagios-metrics-wf.py [PATH_PERFDATA]/service-perfdata \
+                                                           --service \
+                                                           --wf_server [WAVEFRONT_URL] \
+                                                           --csp_api_token [CSP_API_TOKEN] \
+                                                           --csp_base_url [CSP_BASE_URL]
+}
+```
+
+#### Using CSP Oauth
```

Comment:
#### Using the Credentials of a Server to Server OAuth App in VMware Cloud Services